app/assets/javascripts/item_approval.js

```diff
@@ -71,10 +71,14 @@ var itemApproval = (function() {
 
     updateAllApprovedNote: function(item) {
       if(this.allItemsAreApproved(item)) {
-        item.closest('tr.holdings')
-            .prev('tr')
-            .find('[data-behavior="all-approved-note"]')
-            .show();
+        var tableRow =  item.closest('tr.holdings')
+                            .prev('tr');
+
+            tableRow.find('[data-behavior="all-approved-note"]')
+                    .show();
+
+            tableRow.find('[data-behavior="mixed-approved-note"]')
+                    .hide();
```

Comment:
if I'm reading the above correctly, the indentation is misleading.  Lines starting `tableRow` are at the same level of nesting as `var tableRow`, correct?    There's also no particular reason to put the last `.blah()` part on the next line, is there?   Lastly, there's an extra space after `=` in line 74